code/modules/atmos_automation/implementation/digital_pvalves.dm

```diff
@@ -1,96 +1,43 @@
-// --- Pressure Valve Status: Enabled/Disabled ---
-/datum/automation/set_pressure_valve_status
-	name = "Pressure Valve: Status"
-	var/pvalve = null
+// --- Conditional Valve Status: Enabled/Disabled ---
+/datum/automation/set_conditional_valve_status
+	name = "Conditional Valve: Status"
+	var/cvalve = null
 	var/enable = FALSE
 
-/datum/automation/set_pressure_valve_status/Export()
+/datum/automation/set_conditional_valve_status/Export()
 	var/list/json = ..()
-	json["pvalve"]=pvalve
+	json["cvalve"]=cvalve
 	json["enable"]=enable
 	return json
 
-/datum/automation/set_pressure_valve_status/Import(var/list/json)
+/datum/automation/set_conditional_valve_status/Import(var/list/json)
 	..(json)
-	pvalve = json["pvalve"]
+	cvalve = json["cvalve"]
 	enable = text2num(json["enable"])
 
-/datum/automation/set_pressure_valve_status/process()
-	if(pvalve)
-		parent.send_signal(list ("tag" = pvalve, "command" = "enable", "enable" = enable))
+/datum/automation/set_conditional_valve_status/process()
+	if(cvalve)
+		parent.send_signal(list ("tag" = cvalve, "command" = "enable", "enable" = enable))
 	return 0
 
-/datum/automation/set_pressure_valve_status/GetText()
-	// Enable/Disable pressure valve "valve_id"
-	return "<a href=\"?src=\ref[src];set_enable=1\">[enable?"Enable":"Disable"]</a> pressure valve <a href=\"?src=\ref[src];set_subject=1\">[fmtString(pvalve)]</a>"
+/datum/automation/set_conditional_valve_status/GetText()
+	// [Enable|Disable] conditional valve "valve_id"
+	return "<a href=\"?src=\ref[src];set_enable=1\">[enable?"Enable":"Disable"]</a> conditional valve <a href=\"?src=\ref[src];set_subject=1\">[fmtString(cvalve)]</a>"
 
-/datum/automation/set_pressure_valve_status/Topic(href,href_list)
+/datum/automation/set_conditional_valve_status/Topic(href,href_list)
 	if(href_list["set_enable"])
 		enable =! enable
 		parent.updateUsrDialog()
 		return 1
 
 	if(href_list["set_subject"])
-		var/list/pvalves=list()
+		var/list/cvalves=list()
 		for(var/obj/machinery/atmospherics/trinary/pressure_valve/digital/V in atmos_machines)
 			if(!isnull(V.id_tag) && V.frequency == parent.frequency)
-				pvalves |= V.id_tag
-		if(pvalves.len == 0)
-			to_chat(usr, "<span class='warning'>Unable to find any pressure valves on this frequency.</span>")
+				cvalves |= V.id_tag
+		if(cvalves.len == 0)
+			to_chat(usr, "<span class='warning'>Unable to find any conditional valves on this frequency.</span>")
 			return
-		pvalve = input("Select a valve:", "Sensor Data", pvalve) as null | anything in pvalves
-		parent.updateUsrDialog()
-		return 1
-
-// --- Pressure Valve Mode: Threshold and Above/Below ---
-/datum/automation/set_pressure_valve_mode
```

Comment:
Arek Tjallson

Reply:
That is right.
`/datum/automation/` are the commands that show up on the AAC. and `/datum/automation/set_pressure_valve_mode` would be the command I added when I first coded pressure valves to control their settings through the AAC. So yes, deleting that command would result in some feature loss. 
...I should note that in the changelog.

As to what led to this: #33709 brought up a request to 
>allow comparison between the control pipe (the one that is used for the conditional check) and input pipe (the one that is opened to the other side)

And that meant being able to select which input to compare the control pipe against (as there are two inputs that), which in turn meant naming each pipe. 
Applying the North/South/East/West naming scheme used in other atmos components such as gas mixers onto the valve's menu was easy enough... But it did not mesh well when it came to the AAC: as far as I can tell there's no way for me to read the controlled valve's `dir`, nor would it make sense as I must take into account the possibility that the AAC may be controlling multiple valves (each with their own `dir`).

So it's either:
* Adopt a new, `dir`-agnostic naming scheme specifically for conditional valves (my first choice), or
* Drop AAC support entirely (the choice that won out after some brief discussion in this PR)

You may want to look at [3bbba54](https://github.com/vgstation-coders/vgstation13/pull/34418/commits/3bbba54854e09b1067e4b1adca4f023538b14f0b) for a comparison between the two. I am open to alternative solutions as well.

Reply:
Forgot to mention, there is a third option in that I could have the AAC and the valve follow their own naming schemes each, but I really don't like the lack of consistency there.